### ISSUE TITLE
NetAdapterRss Resource Additional Properties

### ISSUE THREAD

**Issue by nmdange2** [PowerShell/NetworkingDsc#367]
### Description
The NetAdapterRss resource only allows enabling/disabling Rss, but Set-NetAdapterRss can set other parameters.  I use some these parameters when configuring VMQ on Hyper-V hosts.

### Proposed properties
NumberOfReceiveQueues
Profile
BaseProcessorGroup
BaseProcessorNumber
MaxProcessorGroup
MaxProcessorNumber
MaxProcessors
NumaNode

### Special considerations or limitations
MaxProcessorNumber and MaxProcessors are interdependent so setting them both could cause conflicting settings.

**Comment by PlagueHO** [comment]
Good idea. If implementing this I'd suggest moving this resource to be data driven - similar to DnsClientGlobalSetting (see https://github.com/PowerShell/NetworkingDsc/blob/dev/Modules/NetworkingDsc/DSCResources/MSFT_DnsClientGlobalSetting/MSFT_DnsClientGlobalSetting.data.psd1).

I'm planning on doing some work on NetworkingDsc over the next few weeks since my most recent PR was completed. This one could be a good one to implement reasonably easily.

**Comment by nmdange2** [comment]
I'd love to implement it, but I only just learned DSC.  I have some learning to do before I can start contributing!

**Comment by PlagueHO** [comment]
No worries @nmdange2 - if you're ever interested in contributing, the community over here are always happy for the help and will support and guide you through the process! We all started at the same place :grin:

**Comment by nmdange2** [comment]
Just wanted to comment on this that due to changes in WS 2019, I don't need these features so much anymore.  I'm ok closing this issue unless someone else wants these features.

https://techcommunity.microsoft.com/t5/Networking-Blog/Synthetic-Accelerations-in-a-Nutshell-Windows-Server-2019/ba-p/653976
